app/views/comparisons/shared/annual_change_in_out_of_hours_use/_table.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= component 'comparison_table',
+              report: report, advice_page: advice_page, table_name: table_name, index_params: index_params,
+              headers: headers, colgroups: colgroups, advice_page_tab: advice_page_tab do |c| %>
+  <% results.each do |result| %>
+    <% c.with_row do |r| %>
+      <% r.with_school school: result.school %>
+
+      <% if result.electricity_economic_tariff_changed_this_year %>
```

Comment:
Same comment about it possibly being a different column for gas. Not sure how storage heaters works though...

Reply:
Storage heaters use the electricity tariffs, gas has its own tariffs, so you need to use the electricity var for storage heater footnotes and the gas var for gas footnotes.

Reply:
corrected